Std/Lean/IO/Process.lean

```diff
@@ -0,0 +1,43 @@
+/-
+Copyright (c) 2023 Scott Morrison. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Scott Morrison
+-/
+
+/-!
+# Running external commands.
+-/
+
+namespace IO.Process
+
+open System (FilePath)
+
+/--
+Pipe `input` into stdin of the spawned process,
+then return `(exitCode, stdout, stdErr)` upon completion.
+-/
+-- TODO We could reduce some code duplication by centralising some functions like this,
+-- which are used here, in `cache`, and in https://github.com/leanprover-community/llm.
+def runCmdWithInput' (cmd : String) (args : Array String)
+    (input : String := "") (throwFailure := true) : IO (UInt32 × String × String) := do
```

Comment:
```suggestion
    (input : String := "") (throwFailure := true) : IO Output := do
```
Isn't there a struct specifically for this?

Reply:
done